helm/alfresco-content-services/README.md

```diff
@@ -151,7 +157,7 @@ Hence, setting up explicit Container memory and then assigning a percentage of i
 | msTeamsService.teams.chat.filenameEnabled | bool | `true` |  |
 | msTeamsService.teams.chat.imageEnabled | bool | `true` |  |
 | msTeamsService.teams.chat.metadataEnabled | bool | `true` |  |
-| ooi | object | `{"enabled":false}` | Choose if you want Office Online Integration capabilities (Alfresco Collaboration Connector for Microsoft 365) |
+| ooi | object | `{"enabled":false,"nodeSelector":{}}` | Choose if you want Office Online Integration capabilities (Alfresco Collaboration Connector for Microsoft 365) |
```

Comment:
this change looks unintended

Reply:
Moved this variable from ooi to ooiService